### PR TITLE
Scholarship: add reimbursement policy

### PR DESCRIPTION
**Pull request by bhavin192**
Preview: https://inpycon2024-hhl6uhkhw-python-indias-projects.vercel.app/scholarship/#reimbursement-policy

### CODE REVIEW

**Review by bhansa — APPROVED** [review]

File: src/scholarship.md [hunk 1]
```diff
@@ -16,3 +16,41 @@ The deadline to apply for the scholarship is 10th Sept, 23:59 IST.
...
+## Reimbursement Policy
+
+- Attendance is required to receive reimbursement. If our ticketing platform does not show you as 'checked-in', you will need to provide proof of attendance.
+- Reimbursement will be up to the amount specified in your acceptance email.
+- Policy violations include (but are not limited to):
+  - Submitting receipts for reimbursement if you did not attend.
+  - Applying with multiple or alias emails.
+  - Any other abuse of the system.
+
+### Receipts for Reimbursement
```
Comment (palnabarun) [inline src/scholarship.md:30]: ### Eligible Receipts for Reimbursement

File: src/scholarship.md [hunk 2]
```diff
@@ -16,3 +16,41 @@ The deadline to apply for the scholarship is 10th Sept, 23:59 IST.
...
+
+**Receipts**:  
+- A scan or photo of the actual receipt/invoice/bill is required.
+- Receipts **must** clearly show the date and amount (and name, if possible).
+- Book refundable travel. Receipts will **not** be reimbursed if you are unable to attend the event and have booked non-refundable travel.
+
+**Accommodation charges**:  
+- A maximum of 3 nights is covered.
```
Comment (palnabarun) [inline src/scholarship.md:38]: - A maximum of 2 nights is covered.

File: src/scholarship.md [hunk 3]
```diff
@@ -16,3 +16,41 @@ The deadline to apply for the scholarship is 10th Sept, 23:59 IST.
...
+- We will reimburse accommodations within the event dates and one day before or after, i.e., from the 20th to the 23rd.
```
Comment (palnabarun) [inline src/scholarship.md:39]: - We will reimburse accommodations within the event dates and one day before or after, i.e., from the 21st to the 23rd or 20th to 22nd.

File: src/scholarship.md [hunk 4]
```diff
@@ -16,3 +16,41 @@ The deadline to apply for the scholarship is 10th Sept, 23:59 IST.
...
+- We will **not** reimburse any services other than the stay.
```
Comment (palnabarun) [inline src/scholarship.md:40]: - We will **not** reimburse any services other than the stay, like hotel wifi, inroom services, beverages etc.

File: src/scholarship.md [hunk 5]
```diff
@@ -16,3 +16,41 @@ The deadline to apply for the scholarship is 10th Sept, 23:59 IST.
...
+
+**Ground transportation charges**:  
+- Bus, metro, taxi, and ride shares to/from the airport, train, or bus station (a total of 4 trips) are eligible for reimbursement.
+- Rides from accommodation to/from the event are not reimbursable.
```
Comment (palnabarun) [inline src/scholarship.md:44]: - Reasonable commute from accommodation to/from the event will be reimbursable.

File: src/scholarship.md [hunk 6]
```diff
@@ -16,3 +16,41 @@ The deadline to apply for the scholarship is 10th Sept, 23:59 IST.
...
+
+**Miscellaneous expenses**:  
+- Any expenses outside the categories mentioned above, such as add-ons, seat upgrades, meals, etc., will **not** be reimbursed.
+
+### Process for Reimbursement
+
+- You will receive an email with instructions for submitting your expense report after the event.
+- The expense report and receipts must be submitted within the timeline provided in the post-event email. If not received within this timeline, reimbursement cannot be processed. You will receive the amount post-approval of your report.
+- **Refer to the scholarship acceptance email for the exact amount covered.**
```
Comment (palnabarun) [inline src/scholarship.md:53]: - **The maximum amount that can be reimbursed is the one mentioned in your scholarship approval email.**

**Pull request merged**